TEP002_model_hdf5.rst

```diff
@@ -0,0 +1,62 @@
+TEP002: Meaningful title
+========================
+
+Status
+======
+
+ **Discussion**
+
+Responsible
+===========
+
+TBD
+
+Branches and Pull requests
+==========================
+
+N/A
```

Comment:
A first implementation for this is in https://github.com/tardis-sn/tardis/pull/435. @unoebauer I can just make a PR to this PR and fill out some of the stuff.